### PR TITLE
added Chinese translation

### PR DESCRIPTION
**Pull request by rockonedege**

### CODE REVIEW

**Comment by iWangJiaxiang** [comment]
Looks great! Hope this PR could be merged ASAP

**Comment by xrh0905** [comment]
Nice work! Hope this PR could be merged ASAP

File: resources/lang/cn/validation.php [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        'string'  => ':attribute 至少 :min 字符.',
+        'array'   => ':attribute 至少包含 :min 元素.',
+    ],
+    'not_in'               => '选中的 :attribute 无效.',
+    'numeric'              => ':attribute 必须是数字.',
+    'present'              => ':attribute 必须存在.',
+    'regex'                => ':attribute 格式无效.',
+    'required'             => ':attribute 必需.',
+    'required_if'          => '当 :other 等于 :value 时，:attribute 必需.',
+    'required_unless'      => 'unless :other is in :values.',
```
Comment (L1cardo) [inline resources/lang/cn/validation.php:73]: 这一行漏了

File: resources/lang/cn/validation.php [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+        'file'    => ':attribute 必须介于 :min and :max KB.',
+        'string'  => ':attribute 必须介于 :min and :max 英文字符.',
+        'array'   => ':attribute 必须包含介于 :min and :max 元素.',
+    ],
+    'boolean'              => ':attribute 值必须是 true 或 false.',
+    'confirmed'            => ':attribute 确认信息不匹配.',
+    'date'                 => ':attribute 不是有效日期.',
+    'date_format'          => ':attribute 不符合格式 :format.',
+    'different'            => ':attribute 与 :other 必须不同.',
+    'digits'               => ':attribute 必须 :digits 位.',
```
Comment (L1cardo) [inline resources/lang/cn/validation.php:37]: `:attribute 必须是 :digits 位数`

这个是不是更好

File: resources/lang/cn/validation.php [hunk 3]
```diff
@@ -0,0 +1,121 @@
...
+    'filled'               => ':attribute 必须有值.',
+    'image'                => ':attribute 必须是图像.',
+    'in'                   => '选中的 :attribute 无效.',
+    'in_array'             => ':other 中不包含 :attribute.',
+    'integer'              => ':attribute 必须是整数.',
+    'ip'                   => ':attribute 必须是有效 IP 地址.',
+    'ipv4'                 => ':attribute 必须是有效 IPv4 地址.',
+    'ipv6'                 => ':attribute 必须是有效 IPv6 地址.',
+    'json'                 => ':attribute 必须是有效 JSON 字符串.',
+    'max'                  => [
```
Comment (L1cardo) [inline resources/lang/cn/validation.php:53]: 这几个全错了，应该是`不能大于`， 不是`必须大于`

**Comment by xinmans** [comment]
cool

**Comment by xinmans** [comment]
Hope this PR could be merged ASAP，Chinese need it！

**Comment by KodeStar** [comment]
Can someone review the conflicts and I'll get this merged

**Comment by rockonedege** [comment]
I'll get it resolved. I'll also include some fixes suggested in this discussion, and create another pr.

**Comment by rockonedege** [comment]
ok, it's resolved and I don't have to create another pr.

**Pull request merged**